Acceleratio.XCellKit/ExcelExport/Chart/Chart.cs

```diff
@@ -0,0 +1,192 @@
+﻿using DocumentFormat.OpenXml;
+using DocumentFormat.OpenXml.Drawing.Charts;
+using DocumentFormat.OpenXml.Packaging;
+using DocumentFormat.OpenXml.Spreadsheet;
+using System.IO;
+using System.Linq;
+using Acceleratio.XCellKit.ExcelExport.Helpers;
+using Chart = DocumentFormat.OpenXml.Drawing.Charts.Chart;
+
+namespace Acceleratio.XCellKit.ExcelExport
+{
+    public class Charts : ReportExport
+    {
+        public Charts(ReportExport export)
+            : base(export)
+        {
+
+        }
+        /// <summary>
+        /// Creates, designes and places chart into a sheet.
+        /// Calls overrideable methods for displaying the chart based on ChartType.
+        /// </summary>
+        public void InsertChartInSpreadsheet(Stream stream)
```

Comment:
You should try to incorporate this logic into the existing classes. I.e. Add a method to the SpreadSheet worksheet class. Just as it has a   public void AddTable(SpredsheetTable table, int columnIndex, int rowIndex), it could have AddChart(SpredsheettChart chart, int columnIndex, int rowIndex). Keep the same level of abstraction in the classes.